app/javascript/dashboard/components/widgets/WootWriter/ReplyBottomPanel.vue

```diff
@@ -87,6 +87,10 @@
         :title="'Whatsapp Templates'"
         @click="$emit('selectWhatsappTemplate')"
       />
+      <video-call-button
```

Comment:
Load only for website inbox and API inbox. We need to get clarity on whether we can send a link to other channels. Until then it is disabled.